Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteQueueProxy.h

```diff
@@ -103,7 +102,7 @@ class RemoteQueueProxy final : public PAL::WebGPU::Queue {
 
     WebGPUIdentifier m_backing;
     Ref<ConvertToBackingContext> m_convertToBackingContext;
-    RemoteDeviceProxy& m_parent;
+    RemoteGPUProxy& m_parent;
```

Comment:
Really? Why? I don't understand.

Reply:
I will probably need to send the memgraphs to explain further

Reply:
Indeed this was just a red hearing in the memgraph, there is no cycle as expected.

But it does mean JS is not collecting the GPU* objects in a sufficiently timely fashion to avoid them accumulating in the GPU process.